test/uk/gov/hmrc/helptosave/services/UserCapServiceSpec.scala

```diff
@@ -171,21 +175,21 @@ class UserCapServiceSpec extends TestSupport with ServicesConfig {
       val userCapService = new UserCapServiceImpl(userCapStore, configOverride)
 
       "allow account creation as long as dailyCap is not reached with no record today " in {
-        val userCap = UserCap(yesterday, 9, 100)
+        val userCap = UserCap(yesterday, dailyLimit, totalLimit)
         mockUserCapStoreGetOne(Some(userCap))
 
         result(userCapService.isAccountCreateAllowed()) shouldBe UserCapResponse()
       }
 
-      "not allow account creation if the dailyCap is reached with record today " in {
-        val userCap = UserCap(today, 9, 100)
+      "allow account creation if the dailyCap is not reached with record today " in {
+        val userCap = UserCap(today, dailyLimit - 1, totalLimit)
         mockUserCapStoreGetOne(Some(userCap))
 
         result(userCapService.isAccountCreateAllowed()) shouldBe UserCapResponse()
       }
 
       "not allow account creation if the dailyCap is already reached" in {
-        val userCap = UserCap(today, 10, 100)
+        val userCap = UserCap(today, dailyLimit, totalLimit)
```

Comment:
Should this be the following to just test the daily cap?
val userCap = UserCap(today, dailyLimit, totalLimit - 1)